package.json

```diff
@@ -33,6 +33,7 @@
   },
   "dependencies": {
     "chalk": "^4.0.0",
+    "cli-truncate": "2.1.0",
```

Comment:
Hey! I was wondering if there's a reason why this dependency was locked to a specific version instead of using `^`?

Reply:
No reason, I'm pretty sure I just installed it with `yarn add`. Thanks for noticing, I'll create a fix for relaxing to `^`.